app/src/main/java/chat/rocket/android/main/ui/Menu.kt

```diff
@@ -0,0 +1,78 @@
+package chat.rocket.android.main.ui
+
+import android.view.Menu
+import android.view.MenuItem
+import chat.rocket.android.R
+
+private const val MENU_SECTION_ONE = 1
```

Comment:
It is better to keep `groupId`s and `itemId`s on `res/values/ids.xml` like: 
```
    <item name="menu_section_one" type="id"/>
    <item name="menu_action_chats" type="id"/>
```
This way we keep the ids unique, and don't risk to have id conflicts when adding more menu items on a Fragment for example.

Looking down, there is no need to add the items on code, we can add the admin panel to the XML and just have directly on `MainActivity`
```
private fun setupMenu(menu: Menu) {
    menu.findItem(R.id.menu_action_admin_panel).isVisible =  permissions.canSeeTheAdminChannel()
}
```

Reply:
Not only for the risks by declaring the ids in the current way, but its more interesting putting them in the `ids.xml`.

Added! 
https://github.com/RocketChat/Rocket.Chat.Android/pull/1570/commits/008f9df4bc22124ae63da5d4397cbf07dce42620